1-mandatory/3-reading-list.js

```diff
@@ -80,19 +93,25 @@ function logBooks() {
 - (Reminder: You must have run `npm install` one time before this will work!)
 */
 
-///// Irianni change function names /////////
+////////// TO DO change function names ////////////
 
 test("books are logged", function () {
   expectLogBooksToLog([
     "The Hobbit by J.R.R. Tolkien",
     "Harry Potter by J.K. Rowling",
+    "Pride and Prejudice by Jane Austen",
+    "All About Love by Bell Hooks",
+    "Rebel Ideas by Matthew Syed",
   ]);
 });
 
 test.only("books read", function () {
```

Comment:
I has the same problem >>>>I think if you run both tests at the same time without using .only, the test will fail because you need to modify the logBooks() function and change the name of one of the functions in the exercise and the test.